<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QMainWindow" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>810</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
  margin: 0;
  width: 100vw;
  height: 100vh;
  background: #ecf0f3;
  display: flex;
  align-items: center;
  text-align: center;
  justify-content: center;
  place-items: center;
  overflow: hidden;
  font-family: poppins;
}

QFrame {
  position: relative;
  border-radius: 20px;
  box-sizing: border-box;
  background: #ecf0f3;
}
QLabel{
font: 14pt &quot;MS Shell Dlg 2&quot;;
}
QLabel, QLineEdit, #QPushButton {
  display: block;
  width: 100%;
  padding: 0;
  border: none;
  outline: none;
}


QLineEdit,QComboBox { 
text-align: left;
font: 14pt &quot;MS Shell Dlg 2&quot;;
  padding: 5px;
  padding-left: 8px;
  height: 32px;
  border-radius: 5px;
margin-top:8px;
}

QPushButton {
font: 13pt &quot;MS Shell Dlg 2&quot;;
  margin-top: 20px;
  background: #ff9900;
  height: 40px;
  border-radius: 5px;
  font-weight: 600;
color: #222;
}
#label {
font: 16pt &quot;MS Shell Dlg 2&quot;;
  margin-top: 10px;
  font-weight: 900;
  color: #ff9900;
  letter-spacing: 1px;
}


QPushButton:hover {
  background: #333;
color:#ff9900;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>25</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Foydalanuvchi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font-weight:bold;
color: rgb(0, 136, 255);</string>
         </property>
         <property name="text">
          <string>Elektro Texnika</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Tovarlar</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget { 
text-align: left;
font: 14pt &quot;MS Shell Dlg 2&quot;;
  padding: 5px;
  padding-left: 8px;
  height: 32px;
  border-radius: 5px;
margin-top:8px;
	background-color: rgb(203, 203, 203);
}
QListWidget{
	background-color: rgb(255, 254, 255);
}
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>TV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Laptop</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mobile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>aksesuar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>maishiy</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Soni:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Narxi:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Nomi:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Buyurma berish</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Kategoriyasi</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>444</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>400</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images.qrc">:/new/prefix1/images/TV.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
